core/src/main/java/io/th0rgal/oraxen/packets/PacketEventsAdapter.java

```diff
@@ -0,0 +1,75 @@
+package io.th0rgal.oraxen.packets;
+
+import com.github.retrooper.packetevents.PacketEvents;
+import com.github.retrooper.packetevents.event.PacketListener;
+import com.github.retrooper.packetevents.event.PacketListenerCommon;
+import com.github.retrooper.packetevents.event.PacketListenerPriority;
+import com.github.retrooper.packetevents.manager.server.ServerVersion;
+import io.th0rgal.oraxen.mechanics.provided.gameplay.efficiency.EfficiencyMechanicFactory;
+import io.th0rgal.oraxen.packets.packetevents.InventoryPacketListener;
+import io.th0rgal.oraxen.packets.packetevents.ScoreboardPacketListener;
+import io.th0rgal.oraxen.packets.packetevents.TitlePacketListener;
+import io.th0rgal.oraxen.packets.packetevents.mechanics.provided.gameplay.efficiency.EfficiencyMechanicListener;
+import io.th0rgal.oraxen.utils.SnapshotVersion;
+
+public class PacketEventsAdapter implements PacketAdapter {
+    private PacketListenerCommon titlePacketListener;
+    private PacketListenerCommon inventoryPacketListener;
+
+    private PacketListenerCommon efficiencyMechanicListener;
+
+    @Override
+    public boolean isEnabled() {
+        return PacketAdapter.isPacketEventsEnabled();
+    }
+
+    @Override
+    public void registerInventoryListener() {
+        inventoryPacketListener = register(new InventoryPacketListener(), PacketListenerPriority.MONITOR);
```

Comment:
should we do a check here just in case it never gets registered twice? well it should never happen bc it only is registered in the onEnable but still